syntax = "proto3";
import "google/protobuf/empty.proto";
service DataService {
    rpc GetCommentsData (google.protobuf.Empty) returns (GetCommentsDataReply) {}
  }



  message GetCommentsDataReply {
    string data = 1;
  }
